test/e2e/storage/utils/utils.go

```diff
@@ -194,22 +206,32 @@ func TestVolumeUnmountsFromDeletedPodWithForceOption(c clientset.Interface, f *f
 		// so wait some time to finish
 		time.Sleep(30 * time.Second)
 	}
+
 	By("Expecting the volume mount not to be found.")
-	result, err = framework.SSH(fmt.Sprintf("mount | grep %s", clientPod.UID), nodeIP, framework.TestContext.Provider)
+	result, err = framework.SSH(fmt.Sprintf("mount | grep %s | grep -v volume-subpaths", clientPod.UID), nodeIP, framework.TestContext.Provider)
```

Comment:
ditto. should volume-subpaths always be here?

Reply:
Also is there a specific reason we have the "-v" version flag?

Reply:
-v means exclude the lines that match.  I think it is fine to exclude subpaths from the results because this line is only checking for regular mounts, not subpath mounts.

Reply:
can we use `--invert-match` instead. I confused it with `-V` which is the `--version` flag

Reply:
done